projects/cdk/schematics/utils/is-ci.ts

```diff
@@ -1,5 +1,8 @@
 /// <reference types="node" />
 
+/**
+ * @deprecated: drop in v5.0
+ */
 export function tuiIsCI(): boolean {
```

Comment:
Would it be equivalent to `process.env.CI` ?

Reply:
yes